src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -185,4 +129,16 @@
       <SymbolsToIndex Include="$(OutputPath)ilmerge\*.pdb" KeepDuplicates="false" />
     </ItemGroup>
   </Target>
+
+  <PropertyGroup Condition="'$(IsBuildOnlyXPLATProjects)' != 'true'">
```

Comment:
nit: use the same property group for both cases, and have the condition only on the NuspecFile property.